packages/wrangler/src/hyperdrive/shared.ts

```diff
@@ -0,0 +1,12 @@
+export function capitalizeScheme(scheme: string | undefined) {
+	let ret = "";
```

Comment:
If the scheme is unknown this will return `""`.  Is that what you want?
I would expect to either throw an error or to just return the `scheme` unaltered.

Reply:
This is only used in displaying some information that should be non-critical, I can have it throw if that would wrangler conventions better, but I think an empty string is fine for this